src/components/Datepicker.vue

```diff
@@ -328,13 +334,28 @@ export default {
         document.removeEventListener('click', this.clickOutside, false)
       }
     },
+    onBlur (e) {
+      if (e.relatedTarget) {
+        this.close()
+      }
+    },
     resetDefaultDate () {
       if (this.selectedDate === null) {
         this.setPageDate()
         return
       }
       this.setPageDate(this.selectedDate)
     },
+    updateDate (event) {
+      let valueDate = this.$refs.inputdatepicker.value
+      let currentYear = this.disabled.from.getFullYear().toString().split('')
+      var regex = new RegExp('^(0[1-9]|[1-2]\\d|3[0-1])\\/(0[1-9]|1[0-2])\\/(19[7-9]\\d|200\\d|20[0-' + currentYear[2] + '][0-' + currentYear[3] + '])$')
```

Comment:
What is this regex supposed to do?
You should comment anything that is not obvious.